lib/google_scraping/dashboard/dashboard.ex

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleScraping.Dashboard do
```

Comment:
Based on our convention, this filename should be plural.
![Screen Shot 2022-05-30 at 10 50 57 AM](https://user-images.githubusercontent.com/6076762/170914029-b2222b73-cf04-4b48-9941-ffdba4503b68.png)